### PR TITLE
chore: a few ruff autofixes

### PR DESCRIPTION
**Pull request by chemelli74**

### CODE REVIEW

File: custom_components/midea_ac_lan/config_flow.py [hunk 1]
```diff
@@ -270,11 +267,13 @@ async def async_step_discovery(
...
+                    str(device_id),
+                    device[CONF_IP_ADDRESS],
                 ):
-                    self.available_device[device_id] = (
-                        f"{device_id} ({self.supports.get(device.get(CONF_TYPE))})"
-                    )
+                    self.available_device[
+                        device_id
+                    ] = f"{device_id} ({self.supports.get(
+                            device.get(CONF_TYPE))})"
```
Comment (Necroneco) [inline custom_components/midea_ac_lan/config_flow.py:276]: Python version 3.11 does not allow new lines in expression parts of non-triple-quoted f-strings

File: custom_components/midea_ac_lan/config_flow.py [hunk 2]
```diff
@@ -196,7 +192,8 @@ async def async_step_login(
                 )
                 if self.cloud is None:
                     raise AbortFlow(
-                        f"Can not get midea cloud: {SERVERS[user_input[CONF_SERVER]]}"
+                        f"Can not get midea cloud: {
+                            SERVERS[user_input[CONF_SERVER]]}",
```
Comment (Necroneco) [inline custom_components/midea_ac_lan/config_flow.py:196]: Python version 3.11 does not allow new lines in expression parts of non-triple-quoted f-strings

**Review by Necroneco — CHANGES_REQUESTED** [review]

File: custom_components/midea_ac_lan/config_flow.py (hunk 1 above)
Comment (Necroneco) [reply custom_components/midea_ac_lan/config_flow.py:276]: Do this in later PR

**Review by Necroneco — APPROVED** [review]

**Comment by chemelli74** [comment]
Nothing to change after merging all previous PRs

**Pull request closed without merging**